main.py

```diff
@@ -4,7 +4,7 @@ def main():
     # make your code below
     # print('Hello World')
     ##############################
-
+    print ('Hello Professor')
```

Comment:
Please find the corrected version of line 7 below.
print ('Hello World')

```suggestion
    print ('Hello World')
```